### PR TITLE
Update EVMC

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

**Comment by gumb0** [comment]
@chfast updated to current EVMC master, waiting for release

**Comment by chfast** [comment]
I believe this should now target `release/1.7` branch, because the `1.7.0` just waits for the EVMC release.

File: libaleth-interpreter/VM.cpp [hunk 1]
```diff
@@ -234,7 +228,8 @@ void VM::fetchInstruction()
 {
     m_OP = Instruction(m_code[m_PC]);
     auto const metric = (*m_metrics)[static_cast<size_t>(m_OP)];
-    adjustStack(metric.num_stack_arguments, metric.num_stack_returned_items);
+    adjustStack(
```
Comment (chfast) [inline libaleth-interpreter/VM.cpp]: This might be handled here: if you check `adjustStack()` it actually reverts the addition.

Comment (gumb0) [reply libaleth-interpreter/VM.cpp]: @chfast Not sure I understand, do you mean changing `adjustStack` to something like this?
```
void VM::adjustStack(int _required, int _change)
{
    m_SP = m_SPP;

    // adjust stack and check bounds
    if (m_stackEnd < m_SP + _required)
        throwBadStack(_required, _change);
    m_SPP += _change;
    if (m_SPP < m_stack)
        throwBadStack(_removed, _added);
}
```

Comment (chfast) [reply libaleth-interpreter/VM.cpp]: Yes.

**Pull request merged**